ffi_tests/generated.h

```diff
@@ -28,9 +28,27 @@ int32_t read_foo (
 void free_foo (
     foo_t * foo);
 
+/** \remark Has the same ABI as `uint8_t` **/
+#ifdef DOXYGEN
+typedef enum Bar
+#else
+typedef uint8_t Bar_t; enum
+#endif
+{
+    /** . */
+    BAR_A,
+}
+#ifdef DOXYGEN
+Bar_t
+#endif
+;
+
+void check_bar (
+    Bar_t _bar);
+
 /** \brief
  *  Concatenate the two input strings into a new one.
- *
+ * 
```

Comment:
Whitespace always finds a way back in

Reply:
Yeah this is on the `master` version which does not have your fix 😄

Reply:
Ah, makes sense! Ignore me :homer_reverse_into_hedge:

Reply:
😆 I really need to get all these branches re-unified 😩